preview/FS-1068-open-type-declaration.md

```diff
@@ -0,0 +1,273 @@
+# F# RFC FS-1068 - Open Type Declaration
+
+The design suggestion [Open static classes](https://github.com/fsharp/fslang-suggestions/issues/383) has been marked "approved in principle".
+This RFC covers the detailed proposal for this suggestion.
+
+
+[Discussion thread](https://github.com/fsharp/fslang-design/issues/352)
+
+* [x] Approved in principle
+* [x] Details: [Resolved to Preview](https://github.com/fsharp/fslang-design/issues/352)
+* [x] Implementation: [Complete to Preview](https://github.com/dotnet/fsharp/pull/6325)
+
+
+## Summary
+[summary]: #summary
+
+Add support for opening types to gain access to static members and nested types, e.g. `open type System.Math`. For example
```

Comment:
This isn't defining what opening a type is. It explains what it can do, but not what it is. We're adding a new kind of open declaration to the language - `open type SomeType`, which allows for unqualified access to its static members and nested types. Once a definition like this is in place then this summary is good.